### PR TITLE
Add tests to travis

### PR DESCRIPTION
**Pull request by slsh1o**

### CODE REVIEW

**Review by webknjaz — CHANGES_REQUESTED** [review]
It will have just one job unless you add some matrix expansion:
https://docs.travis-ci.com/user/build-matrix/#explicitly-included-jobs-with-only-one-element-in-the-build-matrix

Also, the PR title shouldn't end with a period. Plus the commit message should contain the summary, not repeat each and every line of the diff. Implementation details shouldn't be in the commit title — the commit itself already contains the implementation that is perfectly readable. What is the high-level effect of this commit? That's what the commit message title should answer.

**Comment by slsh1o** [comment]
Ok now travis able to run tests but failed with empty secret key. Looks like I should implement #27 first.

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,12 @@
 language: python
-python: 3.8
+python:
+  - 3.8
+  - 3.8
```
Comment (webknjaz) [inline .travis.yml]: Don't use the same version twice

**Review by webknjaz — CHANGES_REQUESTED** [review]
You may use https://docs.travis-ci.com/user/environment-variables/#defining-public-variables-in-travisyml to define some env vars for now.

**Review by webknjaz — APPROVED** [review]

**Pull request merged**